src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureConditions.cs

```diff
@@ -10,5 +10,10 @@ internal class FeatureConditions
     {
         [JsonPropertyName("client_filters")]
         public List<ClientFilter> ClientFilters { get; set; } = new List<ClientFilter>();
+
+        //
+        // TODO: Need a JSON Converter for enum type
```

Comment:
Here I create a new type `RequirementType` as an enum, but looks like there's no built-in JSON converter for enum type.

I find related issues:
- https://github.com/dotnet/runtime/issues/31081
- https://github.com/dotnet/runtime/issues/29975

Not sure whether I should implement a Json converter for enum on my own, or introduce third party packages?

Reply:
Or to use [JsonStringEnumConverter](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonstringenumconverter?view=net-6.0)?

Reply:
> Or to use [JsonStringEnumConverter](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonstringenumconverter?view=net-6.0)?

It seems to work!
I'm not yet familiar with .NET, let me know if there's any consideration about introducing it.